### DISCUSSION TITLE
Remove Tabard of the Argent Dawn from the CC shop if scourge invasion event is planned.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#5723] [Suggestions]
Currently the [Tabard of the Argent Dawn](https://www.chromiecraft.com/product/tabard-of-the-argent-dawn-2/) is being offered on the ChromieCraft rewards catalogue for 10,000 points. This tabard was originally a quest reward from the vanilla scourge invasion before Naxx was released, and later as a vendor item from the WotLK pre-patch scourge invasion event.

If this pre-patch event is planned, I recommend you remove this tabard from the catalogue as soon as possible, seeing as it will be available during the event and is meant to *only* be available during the event. It's not like the [Tabard of the Protector](https://wowgaming.altervista.org/aowow/?item=28788) is available in the catalogue. Of course, if you're not planning on re-creating the event on ChromieCraft, I fully recommend keeping the tabard in the rewards list.

**Comment by Amrx2** [comment]
Hi Bench, good point made and it is definitely something we want to attempt with the introduction of DK's / Wrath pre-patch. Its been removed.

**Reply by heyitsbench** [reply]
> it is definitely something we want to attempt with the introduction of DK's / Wrath pre-patch.

Glad to hear my sniffs may not be going to waste. Thank you. ❤️
